### DISCUSSION TITLE
moving carts in the grid

### DISCUSSION THREAD

**Discussion by macrophone** [FrancescoCeruti/linux-show-player#272] [Q&A] [answered]
Hi,
I didn't find how to move the carts in the grid, I've tried drag n' drop, it doesn't works. Bug ?  

lisp 0.3.1
Debian Jessie / XFCE

**Comment by FrancescoCeruti** [comment, marked as answer]
Hi,
The commands are:
- Move: CTRL + Drag&Drop
- Copy: SHIFT + Drag&Drop

Let me know if it works.

**Comment by macrophone** [comment]
Yes it works great! Thank you, I didn't find the documentation about this.

**Comment by FrancescoCeruti** [comment]
@macrophone that's because (currently) there is no documentation at all.

**Comment by nodiscc** [comment]
> (currently) there is no documentation at all.

I've started something at https://github.com/FrancescoCeruti/linux-show-player/wiki

**Comment by FrancescoCeruti** [comment]
@nodiscc Great! :)

**Comment by macrophone** [comment]
Cool!

Le 24/02/2016 19:10, Francesco Ceruti a écrit :

> @nodiscc https://github.com/nodiscc Great! :)
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/FrancescoCeruti/linux-show-player/issues/7#issuecomment-188386950.

## 

David Nadasi - Administrateur système - Enercoop

Ligne directe : 01 81 80 23 05 / 515
Mail          : david.nadasi@enercoop.org

Enercoop
10 rue Riquet
75019 Paris
http://www.enercoop.fr
